src/components/Dashboard/Dashboard.tsx

```diff
@@ -53,42 +53,47 @@ export const Dashboard = () => {
         <ul className={style.activities}>
           {currentActivities.map(({ timesDone, id, activity }) => (
             <div
-              className={timesDone === 0 ? style.activityZero : style.activity}
+              className={`${timesDone === 0 && style.zero} ${style.activity}`}
               key={id}
             >
               <span className={style.timesDone}>{timesDone}</span>
               <li>{activity}</li>
 
               <button
-                className={style.btnDone}
+                className={`${timesDone === 0 ? style.zero : style.more} ${
+                  style.btnDone
+                }`}
                 type="button"
                 onClick={() => {
                   handleActivityClick(activity);
                 }}
               >
-                ✔️
+                <div className={style.checkMark} />
               </button>
             </div>
           ))}
         </ul>
         {!formVisible && (
-          <button
+          <div
```

Comment:
It is not a good idea to make div interactable. Use `button` when clickable or `a` when it redirects somewhere

https://hidde.blog/more-common-a11y-issues/#heading-5